### PR TITLE
CB-10673 added --force-copying-src option

### PR DESCRIPTION
**Pull request by bso-intel**
This option is need to rescue some Cordova users who has trouble installing two conflicting plugins that use the same source-file target.

### CODE REVIEW

File: doc/plugin.txt [hunk 1]
```diff
@@ -20,7 +20,10 @@ Manage project plugins
         --browserify ................... Compile plugin JS at build time using 
                                          browserify instead of runtime.
 
-    remove <pluginid>|<name> [...] ..... Remove plugins with the given IDs/name.
+        --force-copying-src ............ forces copying source files from the plugin even if the same
```
Comment (nikhilkh) [inline doc/plugin.txt]: I would just name this option --force. This needs to be documented here: https://github.com/apache/cordova-cli/blob/master/doc/readme.md#cordova-plugin-command

File: doc/plugin.txt [hunk 2]
```diff
@@ -20,7 +20,10 @@ Manage project plugins
...
+                                         file already exists in the target directory.
+
+remove <pluginid>|<name> [...] ..... Remove plugins with the given IDs/name.
```
Comment (nikhilkh) [inline doc/plugin.txt]: Indentation issue

**Comment by bso-intel** [comment]
Hi @nikhilkh 
Thanks for reviewing.
I changed all 5 pull requests (CLI, LIB, android, ios, windows) to accommodate your comments using --force.

File: doc/readme.md [hunk 3]
```diff
@@ -315,6 +315,7 @@ cordova {plugin | plugins} [
 |       |--link                   | When installing from a local path, creates a symbolic link instead of copying files. The extent to which files are linked varies by platform. Useful for plugin development.
 |       |--save                   | Save the `<plugin-spec>` as part of the `plugin` element  into `config.xml`.
 |       |--browserify             | Compile plugin JS at build time using browserify instead of runtime.
+|       |--force                  | Forces copying source files from the plugin even if the same file already exists in the target directory.
```
Comment (nikhilkh) [inline doc/readme.md]: One last change - add in italics:
_Introduced in version 6.1_. Forces copying source files....

Also, please merge and re-base all commits - I will be happy to merge this

Comment (bso-intel) [inline doc/readme.md]: Yup, I rebased and changed readme.md as you requested.
Please merge all 5 pull requests in CLI, LIB, ANDROID, IOS, WINDOWS.
Thanks.

**Pull request closed without merging**